### ISSUE TITLE
Update browser-history description

### ISSUE THREAD

**Issue by Samyak2** [pesos/pesos.github.io#59]
**Description**

[browser-history](https://github.com/pesos/browser-history/) has changed quite a bit since v0.0.1 when it was added here. Specifically the code snippet given on the project's page does not work anymore on the latest release.

A detailed learning path is also missing.

Though a minor issue, there is also no direct link to the project's GitHub page.

**Comment by tfidfwastaken** [comment]
Hi, so I am not sure what I should put for a detailed learning path. Could you list the requirements here?

**Comment by Samyak2** [comment]
I was thinking of sending a PR directly. Is that ok?

**Comment by tfidfwastaken** [comment]
Yes, although pull in the small changes that I have made to your page now and then do as you please
